Backpack/src/main/res/values/dimens.xml

```diff
@@ -30,4 +30,6 @@
   <dimen name="bpk_dialog_flare_max_width">440dp</dimen>
   <dimen name="bpk_snackbar_min_height">60dp</dimen>
   <dimen name="bpk_bottom_nav_height">56dp</dimen>
+  <dimen name="bpk_barchart_popup_triangle_width">12dp</dimen>
```

Comment:
Where does this come from? Isn't this pointer supposed to be the "flare" design?

Reply:
From design in Firma. Flare cannot be really used here as the popup is only a drawable and now a view.